src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/VariantAnnotatorIntegrationTest.java

```diff
@@ -364,6 +361,42 @@ public void testDeNovo() {
         Assert.assertEquals(lociWithHighConfidenceDeNovo, new int[] {10130767, 10197999});
     }
 
+    @Test
+    public void addQdToGvsVcf() {
+        final File inputVCF = getTestFile("gvsOutput.vcf");
```

Comment:
Can you rename this to something that gives a little more info about where this is form/why its in the test suite?

Reply:
done